### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by x4d3**
This removes the titles as it's already displayed on the top.
It also add more stuff in the yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
   - ♥️
   - YAML
+when: 2020-12-12T12:12:12
+where:
+  city: Boulder
+  states: CA
+  country: USA
+links:
+  - https://onlineyamltools.com/edit-yaml
+  - https://yaml.org
```
Comment (jennysharps) [inline README.md:15]: Are we sure we want to send people away from our website? Maybe we can link to a second page which redirects them to the website 🌮

Comment (x4d3) [reply README.md:15]: I thought it was ok to link to our friends website. We should now convince them to link to us.

Comment (timstott) [reply README.md:15]: Soon we'll be able to link to our API!!!

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 ---
 yaml_conf: ON
 reasons:
   - I
   - ♥️
   - YAML
+when: 2020-12-12T12:12:12
+where:
+  city: Boulder
+  states: CA
```
Comment (timstott) [inline README.md]: Close enough 😂 
```suggestion
  states: CO
```

**Review by timstott — COMMENTED** [review]
Excellent work

**Pull request merged**